update-reproducibility-summary.sh

```diff
@@ -8,22 +10,22 @@ countGa=0
 countVersion=0
 countVersionOk=0
 
-for buildspec in `find content -name "*.buildspec" -print | sort`
+for buildspec in $(find content -name "*.buildspec" -print | sort)
 do
   ((countVersion++))
   # reset recent fields added to buildspec, to avoid rework of older specs
   diffoscope=
   issue=
 
-  . $buildspec
+  . "$buildspec"
   new_anchor="${groupId}:${artifactId}"
-  [[ -z "${issue}" ]] && [[ ! -z "${diffoscope}" ]] && issue="$(dirname ${buildspec})/$(basename ${diffoscope})"
+  [[ -z "${issue}" ]] && [[ -n "${diffoscope}" ]] && issue="$(dirname "${buildspec}")/$(basename "${diffoscope}")"
 
-  buildinfo="`dirname ${buildspec}`/`basename ${buildinfo}`"
-  if [ `ls ${buildinfo} | wc -l` -le 1 ]; then
+  buildinfo="$(dirname "${buildspec}")/$(basename "${buildinfo}")"
+  if [ $(ls "${buildinfo}" | wc -l) -le 1 ]; then
```

Comment:
ok, I found the real root cause: please remove quotes
`ls "${buildinfo}"` need to be replaced with `ls ${buildinfo}`

Reply:
cool. I made that change in b527dc2, and now I'm seeing a different error (with less output). will see if I can find the cause of that.

Reply:
I think I figured out it is just the last line of the script that is causing a non-zero return value from `update-reproducibility-summary.sh`:

```shell
grep "unexpected issue" README.md
```

Am I correct in thinking we actually "want" that command to "error" out, meaning everything is happy if that command does not find the text "unexpected issue" in README.md?

Reply:
Assuming I'm not totally missing the point (a major assumption), I changed the `grep` command in e7a6495

Reply:
great job, thanks